whylog/teacher/__init__.py

```diff
@@ -91,22 +99,29 @@ def guess_patterns(self, line_id):
         Returns list of guessed patterns for a line.
         """
         pattern_matches = self.pattern_assistant.guess_pattern_matches(line_id)
-        return [pattern_match.pattern for pattern_match in pattern_matches]
+        # TODO: Consider returning dict[int, pattern_match] in order to show guessed converters too.
+        return [pattern_match.pattern for pattern_match in six.itervalues(pattern_matches)]
 
     def choose_guessed_pattern(self, line_id, pattern_id):
         self.pattern_assistant.update_by_guessed_pattern_match(line_id, pattern_id)
 
     def set_pattern_name(self, line_id, name):
```

Comment:
I'm not sure but I thought that Leszek & Paweł on the last meeting said that we don't need that function, and the name of pattern should be generated automaticly.